omod/src/main/webapp/fragments/hiv/programLocationWarning.gsp

```diff
@@ -1,15 +1,18 @@
-
+<% hivProgramLocation.name !=null {%>
 <div>
 <div id="hiv-program-location-info" style="background-color: lightcoral; padding: 10px; border-radius: 5px;">
       <ul>
           <li>Session Location: <b>${sessionLocation.name}</b></li>
           <li>
+             <% if(error !=null){ %>
```

Comment:
here you would check if hivProgramLocation != null and write the error message if necessary